AUTHORS.rst

```diff
@@ -1,35 +1,54 @@
 .. -*- mode: rst -*-
 
-People
+cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+External projects used and papers to cite 
 ------
+sammba-mri is essentially a wrapper for a number of other tools, adapting them 
+for use with small mammal brain MRI. There is not yet a publication about 
+sammba-MRI itself. If you use it, please cite the tool name ``SaMmBa-MRI`` and 
+the website address.
+
+DICOM import uses the dcmdump tool of `OFFIS dcmtk 
+<http://dicom.offis.de/dcmtk.php.en>`_. Export to NIfTI uses `nibabel 
+<http://nipy.org/nibabel/>`_. Understanding the affine was greatly aided by
+reading common.py from `dicom2nifti 
+<http://dicom2nifti.readthedocs.io/en/latest/>`_.
+
+Spatial normalization almost exclusively uses
+`AFNI <https://afni.nimh.nih.gov/>`_, the exception being RATS, which is the 
+usual option for the usually-used brain extraction step. If you do use RATS,
+`cite the RATS reference papers.
+<http://www.iibi.uiowa.edu/content/rats-rapid-automatic-tissue-segmentation-rodent-brain-mri>`_
+File management and interfacing with these tools is handled by nipype (`website
+<http://nipype.readthedocs.io/en/latest/>`_ and `paper
+<https://www.ncbi.nlm.nih.gov/pubmed/21897815>`_).
 
-This work is made available by the MINDt team.
-The DICOM to NIFTI conversion as weel as all the spatial preprocessing
-pipelines have been done by Nachiket Nadkarni, initially in shell. Salma
-Bougacha contributed to recoding them in python.
+Nilearn is the package integrated for analysis of fMRI data. Please read `how to 
```

Comment:
May be you can put 'for analysis of fMRI data and for brain plotting'. All brain plots in the examples rely on nilearn plotting displays.